apps/examples/smartfs/smartfs_powercut/Kconfig_ENTRY

```diff
@@ -1,3 +1,3 @@
 config ENTRY_SMARTFS_POWERCUT
-	bool "Smartfs Powercut Test example"
+	bool "jSmartFs Powercut Test example"
```

Comment:
The TASH command for this test should be changed to "jsmartfs_powercut"

Reply:
changed application name from "smartfs_powercut" to "jsmartfs_powercut" in Makefile